### ISSUE TITLE
Change how to become a partner

### ISSUE THREAD

**Issue by ralphboeije** [part-up/part-up#695]
The idea is to remove the Take Part button and modal with a direct way to propose to become a partner. This is much like the current propose contribution feature on the activities. The latter feature can remain, but the motivation should be posted to a new "Wants to become partner" update.

Given I am a logged in user
And I am on a part-up
When I click the "Become partner" button
Then I see a modal open asking for my motivation
And I can fill in my motivation
And I can click a Share motivation or a No, thanks button

Given I am a logged in user
And I am on the motivation modal
When I click one of the buttons
Then I am on the updates view
And I see the Become partner button in "pending approval" state

**Comment by AriellevdLinde** [comment]
Below you can find the flow for this user story. If there are any questions please let me know. 
![1 become a partner - part-up overview](https://cloud.githubusercontent.com/assets/22590642/19041013/bf98cb52-8986-11e6-9788-9c7d6fd6768e.png)
![2 become a partner - motivation modal](https://cloud.githubusercontent.com/assets/22590642/19041015/bf9d06cc-8986-11e6-9227-fdfe87b13858.png)
![3 become a partner - pending view](https://cloud.githubusercontent.com/assets/22590642/19041016/bfa0ed3c-8986-11e6-9582-2a7ff90c0986.png)
![4 become a partner - pending view for current members](https://cloud.githubusercontent.com/assets/22590642/19041014/bf9c8580-8986-11e6-9d99-e5ec7b8e7d9b.png)

**Comment by ralphboeije** [comment]
Looking good @AriellevdLinde Copy of update could be a bit shorter by just stating "x would like to become partner in this part-up". In the notification the part-up title is always shown.

**Comment by AriellevdLinde** [comment]
Sounds good @ralphboeije, I've changed the copy in the design.

**Comment by AriellevdLinde** [comment]
@andreleon Design is ready

**Comment by prik** [comment]
@ralphboeije and @AriellevdLinde: what should be the flow on accept/decline actions?

**Comment by ralphboeije** [comment]
@prik I think just the same as on contribution accept/decline. See below, user x is current partner, user y requested to become a partner

- Accept:
 - Add <user y> to partners of the part-up
 - Create an update: "<user x> accepted <user y> as partner"
 - Send notification to <user y> and all <users that are partners in this part-up> of this update
 - Do not show button "become partner" 
- Decline:
 - Create an update: "<user x> declined the request of <user y> to become a partner"
 - Send notification to <user y> of this update
 - Toggle state of "become partner" button back to original instead of "pending" for <user y>

**Comment by prik** [comment]
@ralphboeije So you want a new update next to the request update? Or do you want the request update to be changed into the accepted/declined update? Because in you above flow: what would happen to the request update?

**Comment by ralphboeije** [comment]
@prik no, your idea is better. Changing the update also keeps potential communication below the request/accept/decline together.

**Comment by ralphboeije** [comment]
@andreleon on the motivation modal, the cancel link is not needed, as the user can already use the x to close the modal

**Comment by ralphboeije** [comment]
@AriellevdLinde , just found a major loophole in the current working of the my tribes navigation. When you are invited to a part-up and become a partner, you do not automatically become a tribe member (and this is intentional). However, that also means that you do not see the tribe in the My tribes dropdown and you do not see the part-up you just became a partner of. See screenshots for this case at hand.

<img width="1440" alt="screen shot 2016-10-17 at 16 38 05" src="https://cloud.githubusercontent.com/assets/14044235/19442143/9ffea25a-9488-11e6-85f9-7d2dc7461af4.png">
<img width="1440" alt="screen shot 2016-10-17 at 16 38 07" src="https://cloud.githubusercontent.com/assets/14044235/19442144/a014447a-9488-11e6-9e49-41682a9eac58.png">

**Comment by AriellevdLinde** [comment]
@ralphboeije That is indeed a major loophole. Could we solve it with something like "show all part-ups" above discover? Or only part-ups that don't belong to a tribe or where you aren't a member of?

![1 2 logged in - my tribes in header - dropdown tribes all part-ups](https://cloud.githubusercontent.com/assets/22590642/19442696/84eb0c40-948a-11e6-9b79-eab6091e2f51.png)

**Comment by AriellevdLinde** [comment]
@ralphboeije 

I've designed the two scenarios as discussed in the screen below:
- Show the tribes of the part-ups you are a partner/supporter of. Also when you are not a partner of that tribe yet. This will be shown in the dropdown as "not yet a partner".
- The part-ups without a tribe will have there own secondary dropdown list, which will be shown at the bottom of the "my tribes" list. We have to think of the copy, but for now I named it "Part-ups without a tribe"

![1 2 logged in - my tribes in header - dropdown tribes all part-ups](https://cloud.githubusercontent.com/assets/22590642/19444262/0430de62-9490-11e6-8886-ce82f6334f08.png)
